README.md

```diff
@@ -4,103 +4,99 @@ This package provides a set of language helpers, utility functions, and classes
-* `from` - Creates an `Array` from an array-like object
-* `fill` - Fills some or all elements of an array with a given value
-* `find` - Returns the first value in the array satisfying a given function
-* `findIndex` - Returns the first index in the array whose value satisfies a given function
-* `of` - Creates an `Array` with the given arguments as its elements
+The `dojo-core/array` module contains analogues to some of the ES2015 Array APIs.
 
-#### lang
+#### [lang](docs/lang.md)
 
-The `lang` module contains various utility functions for tasks such as copying objects and creating late-bound
-or partial applications of functions.
+The `dojo-core/lang` module contains various utility functions for tasks such as copying objects and creating late-bound
+or partially applied functions.
 
-#### math
+#### [math](docs/math.md)
 
-The `math` module contains analogues to a number of ES2015 APIs, including many trigonometric and logarithmic
+The `dojo-core/math` module contains analogues to a number of ES2015 APIs, including many trigonometric and logarithmic
 functions.
 
-#### string
+#### [string](docs/string.md)
 
-The `string` module contains analogues to the following ES2015 String APIs:
+The `dojo-core/string` module contains analogues to the some of the ES2015 String APIs.
 
-* `codePointAt` - Returns the UTF-16 encoded code point value of a position in a string
-* `endsWith` - Determines whether a string ends with the given substring
-* `fromCodePoint` - Creates a string using the specified sequence of code points
-* `includes` - Determines whether a string includes the given substring
-* `repeat` - Returns a string containing a string repeated a given number of times
-* `startsWith` - Determines whether a string begins with the given substring
+#### [UrlSearchParams](docs/UrlSearchParams.md)
 
-Special thanks to Mathias Bynens for granting permission to adopt code from his
-[`codePointAt`](https://github.com/mathiasbynens/String.prototype.codePointAt),
-[`fromCodePoint`](https://github.com/mathiasbynens/String.fromCodePoint), and
-[`repeat`](https://github.com/mathiasbynens/String.prototype.repeat) polyfills.
+The `dojo-core/UrlSearchParams` class can be used to parse and generate URL query strings.
 
-The `string` module also contains the following utility functions:
+#### [Event handling](docs/events.md)
 
-* `escapeRegExp` - Escapes a string to safely be included in regular expressions
-* `escapeXml` - Escapes XML (or HTML) content in a string
-* `padEnd` - Adds padding to the end of a string to ensure it is a certain length
-* `padStart` - Adds padding to the beginning of a string to ensure it is a certain length
+The `dojo-core/on` module contains methods to handle events across types of listeners.  It also includes methods to handle different event use cases including only firing
+once and pauseable events.
 
-#### UrlSearchParams
+#### [HTTP requests](docs/requests.md)
```

Comment:
If this is just going to link to a doc about `request`, call the doc `request.md` and make the heading `request`.